app/controllers/state_file/questions/nj_ineligible_property_tax_controller.rb

```diff
@@ -7,6 +7,14 @@ class NjIneligiblePropertyTaxController < QuestionsController
       helper_method :on_home_or_rental
 
       def determine_reason
+        if Efile::Nj::NjPropertyTaxEligibility.ineligible?(current_intake)
+          if current_intake.filing_status_single? || current_intake.filing_status_mfs?
+            return "income_single_mfs"
+          else
+            return "income_mfj_qss_hoh"
+          end
+        end
+
```

Comment:
[dust] - this seems like the cleanest way to do this for this PR, but I do wonder long-term if `determine_reason` would be better living in the `NjPropertyTaxEligibility` util... though that refactor would be good to do alongside combining with the `NjHomeownerEligibilityHelper` and `NjTenantEligibilityHelper` into one central brain for property tax eligibility. but again, probably for another day.

Reply:
linking to the ticket which captures this work for cross-reference

https://github.com/newjersey/affordability-pm/issues/227